substrate/frame/support/procedural/src/construct_runtime/expand/metadata.rs

```diff
@@ -147,6 +159,12 @@ pub fn expand_runtime_metadata(
 							>(),
 						event_enum_ty: #scrate::__private::scale_info::meta_type::<RuntimeEvent>(),
 						error_enum_ty: #scrate::__private::scale_info::meta_type::<RuntimeError>(),
+					},
+					view_functions: #scrate::__private::metadata_ir::RuntimeViewFunctionsIR {
```

Comment:
I'm still thinking we could put the pallet specific view functions into the `pallets` metadata instead of here? I geuss one reason to haev them here is the idea that there could be runtime-wide view functions added next, but even with that in mind I'd still probably separate them so that in each pallet we can see the pallet specific view functions, and at the top level here we'd see any runtime-wide view functions. What do you think @ascjones?

In any case, I don't think this blocks merging the PR; just something on my mind, and I'm aware that I might be misunderstanding something with this thought :)